include/deployer.bash

```diff
@@ -242,14 +262,15 @@ main() {
         cmd-export deployer-regenerate regenerate
         cmd-export deployer-delete delete
         cmd-export compose-ps ps
-        cmd-export compose-up start
+        cmd-export compose-up-cmd start
+        cmd-export start-and-migrate-cmd start-and-migrate
```

Comment:
@akanto we have discussed with @schfeca75 that we have reached a point where 

- config-xxx and generate and similar must be ironed. sometimes config is called 3-4 times, generate called 2-3 times
- migrate should be "smart". Right now it takes 12 sec to realize that it has nothing to do. (for example you are developing and you issue start/migrate 20 times a day on the same db).

We come up with an idee that we could check that the db is migrated to a specific point by:
```
docker exec -it cbreak_cbdb_1 \
  psql -U postgres -t -c 'select count(*) from changelog where id=20150712112355;'
```

But lets discuss these items in the office.